crates/starknet/src/starknet/mod.rs

```diff
@@ -419,6 +525,28 @@ impl Starknet {
         add_invoke_transaction::add_invoke_transaction(self, invoke_transaction)
     }
 
+    /// save starknet transactions to file
+    pub fn dump_transactions(&self) -> DevnetResult<()> {
+        match &self.config.dump_path {
+            Some(path) => {
+                let starknet_dump = Some(
+                    serde_json::to_string(&self.transactions)
+                        .expect("Failed to serialize starknet transactions"),
+                );
+                let encoded: Vec<u8> = bincode::serialize(&starknet_dump)
```

Comment:
Expect

Reply:
Fixed in 27ec8cc63261fb9430c873a0adda2ceebf38f005